AbstractEntityValidation.md

```diff
@@ -0,0 +1,75 @@
+Feature: Validation of an abstract entity
+
+Scenario: Validation of an abstract entity
```

Comment:
Suggestions:

1. Describe what we do not want to occur: "Ensure that clients cannot POST an abstract entity"
2. Add a line between the Scenario and the `When` statement.

Reply:
Description added and line added between scenario and `when` statement